.github/workflows/sarifdemo.yml

```diff
@@ -1,25 +1,48 @@
-name: "Run Anchore Scan Action (ACS SARIF Demo)"
+name: "[SARIF] Run Scan Action"
 
 on: [push, pull_request]
 
 jobs:
-  demo:
+  sarif-image:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout the code
       uses: actions/checkout@v2
 
-    - name: Run the local anchore scan action itself with sarif generation enabled
+    - name: Run the local Scan Action with SARIF generation enabled
       uses: ./
       with:
-        image-reference: "debian:8"
+        image: "debian:8"
         debug: true
         acs-report-enable: true
-        #acs-report-severity-cutoff: "Medium"
+        #severity-cutoff: "Medium"
+
+    - name: Inspect Generated SARIF
+      run: cat results.sarif
+
+    - name: Upload SARIF
+      uses: github/codeql-action/upload-sarif@v1
+      with:
+        sarif_file: results.sarif
+
+  sarif-directory:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2
+
+    - name: Run the local Scan Action with SARIF generation enabled
+      uses: ./
+      with:
+        path: "tests/python"
+        debug: true
+        acs-report-enable: true
+        #severity-cutoff: "Medium"
+
     - name: Inspect Generated SARIF
       run: cat results.sarif
-# Uncomment this section to enable report upload
-#    - name: Upload Anchore Scan SARIF
-#      uses: github/codeql-action/upload-sarif@v1
-#      with:
-#        sarif_file: results.sarif
+
+    - name: Upload SARIF
```

Comment:
@alfredodeza - I believe this section should remain commented out ( or removed ), as when enabled the sarif results of the test scan will be uplaoded and associated with the security code scanning section of the anchore action repo itself (where the items being scanned are just for testing, and the results will not be correct)

Reply:
Ah, very good point @nurmi! I did find a bunch of issues because of this one test, so I'll create a separate project that uses `scan-action` so that we can run this every night, uploading the SARIF report.

Reply:
This is now commented out